builds/service/service-deployment.yaml

```diff
@@ -0,0 +1,80 @@
+# Run a simple smoke test against a dogfood deployment of the IoT Hub service
+
+pr: none
+trigger: none
+
```

Comment:
Suggest to add this to avoid download source code from repo since this build is not required.
```suggestion
variables:
  Build.SyncSources: false
```

Reply:
Good catch...I forgot to migrate that setting. But wouldn't `- checkout: none` by more appropriate? Docs say that `Build.SyncSources` is specific to TFS2017, not Azure Pipelines.

Reply:
Good to know.  I will update stress and long haul yaml as well. 👍